stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/ChatUI.kt

```diff
@@ -28,6 +28,10 @@ import io.getstream.chat.android.ui.common.BuildConfig
  * @see ChatStrings
  * @see ChatFonts
  */
+@Deprecated(
```

Comment:
This class can't be deprecated on this package.
It should be moved to both UI lib, keep it as it is on the Old UI and do the deprecation process on the new UI lib